OpenRA.Mods.Cnc/Traits/Render/WithCargo.cs

```diff
@@ -45,6 +45,19 @@ public WithCargo(Actor self, WithCargoInfo info)
 			body = self.Trait<IBodyOrientation>();
 		}
 
+		public void Tick(Actor self)
+		{
+			if (facing == null)
+				return;
+
+			foreach (var c in cargo.Passengers)
+			{
+				var cargoFacing = c.TraitOrDefault<IFacing>();
```

Comment:
Tried it in https://github.com/Mailaender/OpenRA/tree/with-cargo-perf but somehow the turrets vanish.